README.md

```diff
@@ -101,22 +101,22 @@ In your app:
 
 ```
 import PlaidLink from 'react-native-plaid-link-sdk';
- 
+
 const MyPlaidComponent = () => {
   return (
     <PlaidLink
-      title='Add Account'
+      component={() => {return <Button title="Add Bank" />;}}
```

Comment:
Passing a `component` is optional. The default `TouchableOpacity` prop is defined here:
https://github.com/plaid/react-native-plaid-link-sdk/blob/master/PlaidLink.js#L145

But if component is not passed, the default TouchableOpacity would need a child `<Text>` element, which we didn't have. Would you like me to update the PR? Or would you like to add that instead?

Reply:
Ah I totally missed the `PlaidLink.defaultProps` declaration, my bad. I'm newer to React in general.

I won't be able to update this PR for a bit, so I suggest you run with any changes you see fit if you have the time and interest. You can close this PR or add to it, your choice. The contribution credit isn't important to me. If you haven't gotten to this by the time I have a chance, I'll update the PR.

Thanks, @wkiefer!

Reply:
@wkiefer can you share working example or Update Readme file. So I can implement it in our application.
I added <Text> as Child but still nothing showing.